packages/components/psammead-brand/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-brand",
-  "version": "7.1.0",
+  "version": "7.2.0",
```

```suggestion
  "version": "7.1.1",
```
I'd say this is fixing a bug in a broken minor version so maybe a patch bump?